components/brave_rewards/resources/page/components/connect_wallet_modal.style.ts

```diff
@@ -214,33 +218,116 @@ export const providerButtons = styled.div`
   }
 `
 
+export const grid = styled.div`
+/* Layout */
+  display: grid;
+  width: 100%;
+/* Grid */
+  grid-template-areas: "icon name caret";
+  grid-template-columns: auto 1fr auto;
+
+  .grayed-out & {
+  /* Grid */
+    grid-template-areas:
+      "icon name"
+      "icon message";
+    grid-template-columns: auto 1fr;
+    grid-template-rows: auto;
+  }
+`
+
 export const providerButtonIcon = styled.div`
-  flex: 0 0 52px;
-  background: rgb(218, 220, 232, .33);
-  border-radius: 50%;
+/* Layout */
+  align-self: center;
+  display: flex;
   height: 52px;
   width: 52px;
-  padding: 14px 0;
+/* Appearance */
+  background: var(--brave-palette-neutral000);
+/* Grid */
+  grid-area: icon;
+/* Other */
+  border-radius: 50%;
 
   .icon {
+  /* Layout */
     height: 25px;
-    width: auto;
+    margin: auto;
+  }
+
+  .grayed-out & {
+  /* Appearance */
+    background: var(--brave-palette-white);
   }
 `
 
 export const providerButtonName = styled.div`
-  flex: 1 1 auto;
-  padding-left: 30px;
+/* Layout */
+  align-items: center;
+  display: flex;
+  padding-left: 14px;
+/* Text */
+  font-size: 16px;
+  font-weight: 600;
+  line-height: normal;
   text-align: left;
+/* Appearance */
+  color: var(--brave-palette-black);
+/* Grid */
+  grid-area: name;
+
+  .grayed-out & {
+  /* Layout */
+    align-items: end;
+  /* Appearance */
+    color: var(--brave-palette-neutral700);
+  }
+`
+
+export const providerButtonMessage = styled.div`
+/* Layout */
+  align-items: start;
+  display: flex;
+  padding-left: 14px;
+/* Text */
+  font-size: 11px;
+  font-weight: 400;
+  line-height: normal;
+  text-align: left;
+/* Appearance */
+  color: var(--brave-palette-neutral700);
+/* Grid */
+  grid-area: message;
 `
 
 export const providerButtonCaret = styled.div`
-  flex: 0 0 10px;
+/* Layout */
+  display: flex;
+/* Grid */
+  grid-area: caret;
 
   .icon {
+  /* Layout */
     height: 16px;
-    width: auto;
-    vertical-align: middle;
-    margin-bottom: 3px;
+    margin: auto;
+  /* Other */
+    fill: var(--brave-palette-blurple500);
+    stroke: var(--brave-palette-blurple500);
+  }
+`
+
+export const learnMoreLink = styled.a`
```

Comment:
I'd like to stick to only styling `div` and `span`. When we style interactive elements, it becomes difficult to tell the difference between the styling aspects of the layout and the functional aspects of the layout.

Instead you can make this style a `div` or `span` and then have a rule inside of it for `a {}`.